loader/src/packfile_io.rs

```diff
@@ -269,7 +299,12 @@ impl LoaderIO for PackfileReader {
         }
     }
 
-    fn tick(&mut self, _loader: &mut LoaderState) {}
+    fn tick(&mut self, _loader: &mut LoaderState) {
+        // We require this yield if the runtime is a CurrentThread runtime
```

Comment:
If we use the multithreaded runtime on a machine with only 1 CPU core, will it have the same problem? Or does it have a thread pool separate from the user's code?

Reply:
It'll have a separate thread pool